tools/buildgen/plugins/expand_version.py

```diff
@@ -127,3 +127,5 @@ def mako_plugin(dictionary):
         else:
             settings[version_tag] = Version(version_str,
                                             override_major=override_major)
+    settings['protobuf_major_minor_version'] = (
+            '.'.join(settings['protobuf_version'].split('.')[:-1]))
```

Comment:
nit: change the `-1` index to `2` here to be more explicit about capturing these two numbers?